bumblebee_status/modules/contrib/shell.py

```diff
@@ -40,6 +40,7 @@ class Module(core.module.Module):
     def __init__(self, config, theme):
         super().__init__(config, theme, core.widget.Widget(self.get_output))
 
+        self.__unit= self.parameter("unit", 'echo "no unit configured"')
```

Comment:
I think you don't want to use "echo..." as default (this is done in the command to provide a meaningful default command)

Probably using None would be best, as you are checking for that later. Or possibly an empty string to make the concatenation work nice in the output.